packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerHeader.tsx

```diff
@@ -20,8 +20,8 @@ export function MomentTemplateBannerHeader({
         <div css={styles.container}>
             <header css={styles.header}>
                 <h2>
-                    <Hide below="tablet">{mobileHeading ?? heading}</Hide>
-                    <Hide above="tablet">{heading}</Hide>
+                    <Hide above="tablet">{mobileHeading}</Hide>
+                    <Hide below="tablet">{heading}</Hide>
```

Comment:
This is actually a bug in the moment template banner atm! We're rendering them the wrong way round